src/Features/Core/Portable/CodeFixes/RoslynAssemblyAttribute.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System;
+
+namespace Microsoft.CodeAnalysis
+{
+    [AttributeUsage(AttributeTargets.Assembly)]
+    internal class RoslynAssemblyAttribute : Attribute { }
```

Comment:
for telemetry, we probably need this whenever we let user to plug in. right now, I believe we have only diagnostic analyzer and code fixer, but I heard we are thinking opening more like intellisense, diagnostic tagger and etc? so might be better to have all roslyn assemblies have this so all other features could just use this method to determine whether it can log data without obfuscation. 

by the way, I think rather than "RoslynAssemblyAttribute", it might be better to be "MicrosoftAssembly" or something like that, so that we can log telemetry for other microsoft component that plug into our stuff like azure tool and etc.